src/mapping/PetRadialBasisFctMapping.hpp

```diff
@@ -1355,13 +1357,20 @@ PetRadialBasisFctMapping<RADIAL_BASIS_FUNCTION_T>::bgPreallocationMatrixC(mesh::
           distance[d] = 0;
 
       double const norm = distance.norm();
+      if (inVertex.getPatchID() == vj.getPatchID()){
+        PRECICE_INFO("ID numbers of vertices: inVertexID = " << inVertex.getID() << " and vjID = " << vj.getID() );
+        PRECICE_INFO("Patch ID matches: inVertexPatchID = " << inVertex.getPatchID() << " and vjPatchID = " << vj.getPatchID() );
```

Comment:
These should probably be `Debug` statements, but we probably also don't want all of this in the end.

Reply:
Agreed. I put them in when checking if the patches were selected correctly. Same for the comment below.